client/ayon_core/hosts/maya/plugins/publish/extract_layout.py

```diff
@@ -145,6 +145,11 @@ def process(self, instance):
 
             json_data.append(json_element)
 
+        shift_back_to_frame_start = instance.data.get("shift_back_to_frame_start", False)
+        if shift_back_to_frame_start:
+            shot_node = instance.data.get("shot_node")
+            # cmds.setAttr(shot_node + ".sequenceStartFrame", instance.data.get("frameStart"))
```

Comment:
```suggestion
```

Reply:
@antirotor is it the right approach to set the shot node start frame like this?

I noticed in the code for extract layout it extracts a json, and doesn't seem to concern itself with the frames of the sequence at all. So offsetting the shot node start frame probably will make no difference.

Reply:
@antirotor any comment?

NOTE: This line should actually not be commented out. Its my proposed solution for offseting the clip back to frame start.

Reply:
> @antirotor any comment?
> 
> NOTE: This line should actually not be commented out. Its my proposed solution for offseting the clip back to frame start.

If it is affected the layout workflow, you can just ignore my comment:)